frontend/src/components/details-screen/detailsScreen.js

```diff
@@ -0,0 +1,229 @@
+// React imports
+import React from 'react'
+
+// Third imports
+import {
+  FaStar,
+  FaSuitcase
+} from 'react-icons/fa'
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+
+// Local imports
+import '../../static/css/details-screen.scss'
+import Tv from '../../static/images/tv.png'
+import Beto from '../../static/images/beto.png'
+import Betinho from '../../static/images/betinho.png'
+import Girl from '../../static/images/girl.png'
+import Gil from '../../static/images/gil.png'
+
+
+export default function DetailsScreen() {
+
+  const options = {
+    chart: {
+      styleMode: true,
+      backgroundColor: 'transparent',
+      marginTop: 5,
+      height: 300,
+      width: 600,
+      spacingLeft: 10
+    },
+    title: '',
+    xAxis: {
+      lineColor: '#fff',
+      lineWidth: 1,
+      categories: ['Jan', 'Feb', 'Mar', 'Apr', 'Mai', 'Jun', 'Jul', 'Ago'],
+      labels: {
+        style: {
+          color: '#fff'
+        }
+      },
+    },
+    yAxis: {
+      gridLineWidth: 0,
+      lineColor: '#fff',
+      lineWidth: 1,
+      min: 500,
+      max: 1700,
+      title: {
+        enabled: false
+      },
+      labels: {
+        style: {
+          color: '#fff'
+        }
+      },
+    },
+    series: [{
+      color: '#B1E1FC',
+      type: 'column',
+      data: [1600, 700, 1200, 980, 1000, 960, 1100, 1600]
+    }],
+    legend: '',
+    credits: {
+      enabled: false
+    },
+    plotOptions: {
+      series: {
+        pointWidth: 17,
+        borderWidth: 0,
+      }
+    }
+  }
+
+  const optionsPie = {
+    series: [{
+      data: [
+        {
+          y: 10,
+          color: '#f9a94b'
+        },
+        {
+          y: 10,
+          color: '#eb6666'
+        },
+        {
+          y: 10,
+          color: '#43db9b'
+        },
+        {
+          y: 10,
+          color:  '#52B6EE'
+        },
+      ]
+    }],
+    chart: {
+      type: 'pie',
+      backgroundColor: 'transparent',
+      height: 210,
+      width: 200,
+      spacingLeft: 10
+    },
+    plotOptions: {
+      pie: {
+        size: 200,
+        borderWidth: 0,
+        dataLabels: {
+          enabled: false,
+        },
+      },
+    },
+    title: '',
+    credits: {
+      enabled: false
+    },
+  }
+
+  const renderStars = () => {
+    let elements = []
+
+    for (let i = 0; i < 5; i++) {
+      elements.push(
+        <FaStar
+          key={i}
+          color='#ffc107'
+          size={14} />
+      )
+    }
+
+    return elements
+  }
+
+  return (
+    <div className='container-main'>
+      <div className='container-graphic'>
+        <p className='title-graphic'>Quantitativo de vendas por mês</p>
```

Comment:
So, if you use the property of highchart that set the title of graph, you don't need use this p tag